packages/sources/polygon-test/src/endpoint/conversion.ts

```diff
@@ -0,0 +1,118 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { buildUrlPath } from './utils'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { customSettings } from '../config'
+import { makeLogger } from '@chainlink/external-adapter-framework/util/logger'
+
+const logger = makeLogger('Polygon Conversion Logger')
+
+export const inputParameters = {
+  base: {
+    aliases: ['from'],
+    required: false,
+    description: 'The symbol of the currency to query',
+    type: 'string',
+  },
+  quote: {
+    aliases: ['to'],
+    required: false,
+    description: 'The symbol of the currency to convert to',
+    type: 'string',
+  },
+  amount: {
+    required: false,
+    description: 'The amount of the `base` to convert ',
+    default: 1,
+    type: 'number',
+  },
+  precision: {
+    required: false,
+    description: 'The number of significant figures to include',
+    default: 6,
+    type: 'number',
+  },
+} as const
```

Comment:
After the typescript upgrade and the fw bump past 0.25, we want to move to using the satisfies operator here like 
```suggestion
} satisfies InputParameters
```

Reply:
I think this is still missing